app/Console/Commands/Init.php

```diff
@@ -148,19 +155,25 @@ protected function fillResources(): void
         $filePart = $this->loadReadmePart('RESOURCES.md');
 
         foreach (self::RESOURCES_ITEMS as $key => $title) {
-            if ($this->confirm("Are you going to use {$title}?", true)) {
-                $defaultLink = ($key === 'telescope') ? $this->appUrl . '/telescope' : '';
+            $defaultAnswer = ($key === 'telescope') ? $this->appUrl . '/telescope' : 'later';
+            $text = "Are you going to use {$title}? " .
+                "Please enter a link or select `later` to do it later, otherwise select `no`.";
```

Comment:
```suggestion
                "Please enter a link or type `later` to do it later, otherwise select `no`.";
```

Reply:
@DenTray As you missed we are using `anticipate()` here. So **select** is the more appropriate word.